src/main/java/se/sundsvall/casedata/api/model/Errand.java

```diff
@@ -71,8 +71,8 @@ public class Errand {
 	@Size(max = 255)
 	private String phase;
 
-	@Schema(description = "The current status of the errand", example = "Pågående", maxLength = 255)
-	private String status;
+	@Schema(description = "The current status of the errand", maxLength = 255)
+	private Status status;
```

Comment:
Status.dateTime should be read only right?

Reply:
And Status.statusType should have nullable = false. While Errand.status still has nullable = true (default)

Reply:
yes. I think we should rename it created - dateTime is a garbage name, set it as readonly and make sure it's populated on prepersist?